src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -931,6 +931,30 @@ format layout_optimizer::get_expected_format(convolution_node const& node) {
         return format::bfyx;
     }
 
+    if (node.is_dynamic() && use_onednn_impls && onednn_valid_post_ops &&
+        output_layout.get_partial_shape().size() == 4 && !i8_u8_input && !node.has_padded_dependency() &&
+        input_layout.get_partial_shape()[1].is_static() && output_layout.get_partial_shape()[1].is_static()) {
```

Comment:
Do we have any reference where we find that these conditions come from? 
e.g., onednn's condition that does not use jit::ir but ref kernel.

Reply:
As discussed, updated condition leaving only the minimum thing